content/blog/updating-to-cpp11/index.md

```diff
@@ -0,0 +1,60 @@
+
+To get started, I followed the initial set-up steps outlined in Jim's article and then recompiled, to confirm it was successful. Next, I needed to include either the `cpp11/R.hpp` header or the macros `R_NO_REMAP` and `STRICT_R_HEADERS`. I found it easier to include the macros since the header file needs to come before any headers that use Rcpp.
+
+## Step-by-step integration
+
+After I'd set-up my environment it was time to start converting the files. Initially, I made the mistake of selecting the most interconnected file in the readxl package. Since this file was central to the package, I would have had to convert the entire package over to cpp11 before the compilation errors would stop. After Jenny and Jim helped me select a more approachable file, the process was more incremental and I successfully maintained my sanity 🙌.
+
+After I finished converting a file, I recompiled, ran the tests for the readxl package and fixed any compilation errors or test failures. Sometimes I had to edit other functions in other files, but I only edited files enough to fix the failures. If I did too much it was difficult for me to know what was causing the test failures. Another incremental approach I used was to convert one function in a file, and then rerun the tests. This approach worked best for larger files. Once the tests were passing, I'd commit and push to the PR so that my gracious reviewers could review my changes. This helped me maintain continuous integration and also helped my mentors review my code in stages, rather than in one, big batch. It's a win-win!
+
+## The nitty-gritty
+
+For the nitty-gritty details of converting the code, I definitely took advantage of [the table in Jim's post](https://cpp11.r-lib.org/articles/converting.html#class-comparison-table-1)! That comparison table is great for converting between Rcpp and cpp11 classes. Also, I had to keep track of whether an object was writable or readable. This new feature in cpp11 is great, since writable vectors are costly because the data must be fully copied, so using readable where appropriate is a good idea. When I was unsure of whether an object should be readable or writable, I would make it readable and then recompile to see if I was correct.
+
+## Almost done
+
+<<<<<<< HEAD
+Then, when I converted all the objects to cpp11, I removed any stray `#include "Rcpp.h"` directives and ran `devtools::check()` to check for any other updates that might be required, such as in the DESCRIPTION file. And that's it!
+=======
+Then, when I converted all the objects to cpp11, I removed any stray #include "Rcpp.h" directives and ran devtools::check() to check for any other updates that might be required, like in the DESCRIPTION files. And that's it!
+>>>>>>> 2a8dd80f6405657c0656789f0a771948f59732cc
```

Comment:
This is some leftover cruft from a merge conflict. Presumably a simple knit of `index.Rmd` will overwrite this whole `.md` file and take care of the problem.